src/map/Map.js

```diff
@@ -611,6 +612,8 @@ L.Map = L.Evented.extend({
 		if (this.options.trackResize) {
 			L.DomEvent[onOff](window, 'resize', this._onResize, this);
 		}
+
+		this[onOff]('moveend', this._onMoveEnd);

```

Comment:
> Not sure what you mean by _leaflet_pos changes.

`initEvents` is called only once at map init, right? But `map._mapPane._leaflet_pos` value is changed at every `_resetView` call, so we cannot rely on its value at init time. Correct?